application/common/models/Password.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace common\models;
+
+use Closure;
+use common\helpers\MySqlDateTime;
+use Yii;
+use yii\behaviors\AttributeBehavior;
+use yii\db\ActiveRecord;
+use yii\helpers\ArrayHelper;
+
+class Password extends PasswordBase
+{
+    public $password;
+
+    public function rules(): array
+    {
+        return ArrayHelper::merge([
+            [
+                'created_utc', 'default', 'value' => MySqlDateTime::now(),
+            ],
+            [
+                'expiration_utc', 'default', 'value' => MySqlDateTime::now(),
```

Comment:
yes, down in `behaviors()`.  Since they are required (from the base) they need to be populated to make it through validation.